Touhou Launcher/Touhou Launcher.csproj

```diff
@@ -137,8 +126,6 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="jsonnet\Newtonsoft.Json.dll" />

```

Comment:
Does `Newtonsoft.Json` have a statically linked library now? The lack of one was why I used this hack to include the dynamically linked library in the executable, as I'd much rather the executable be standalone.

Reply:
If you are asking whether it has a NuGet package, yes.

Reply:
Static library as in, a library that's linked at compile time and doesn't need to be included with the executable as a dll file.

Reply:
Well, no, but .NET 6.0 allows single-file publishing, and it works on Windows 7, so I could migrate the project to that.

Reply:
I see. In that case I'd recommend reverting the change for `Newtonsoft.Json` here and simply embed an updated dll for now. The port to another platform can wait until another pull request.